Lesson21/Kadir/app.js

```diff
@@ -17,35 +17,64 @@ window.addEventListener('load',function(){
   // Show images function
   function showImage(index) {
     // Remove active class from all images
-    images.forEach(
-      // For every element in images array call a function that removes "active" class
-    )
+        images.forEach(element => {
+          element.classList.remove("active");
+        });
+
+        // indicators.forEach(el => {
+        //   el.classList.remove("active");
+        // });
+
+        while(indicators.length>0){
```

Comment:
I'd expect this code to get into an infinite loop as it should keep running as long as you have indicators. Why not use forEach instead?